blueprints/locale/index.js

```diff
@@ -3,23 +3,20 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 
+var broccoliCLDR = require('../../lib/broccoli-cldr');
 var Blueprint   = require('ember-cli/lib/models/blueprint');
 var SilentError = require('ember-cli/lib/errors/silent');
 var Promise     = require('ember-cli/lib/ext/promise');
 var path        = require('path');
 var fs          = require('fs');
 
-var extract     = require('../../lib/extract');
-
 module.exports = {
     description: 'Extract a CLDR data into an ES6 module for a given locale code',
 
-    anonymousOptions: ['fields'],
-
     normalizeEntityName: function (localeName) {
```

Comment:
Correct, spits out an empty translation file which extends from `Locale` class.  I am really considering naming the locale files `translations`, because that's all we store in them at the moment.  It's clearly confusing :)